Tests/ChessGameTest.php

```diff
@@ -1105,4 +1105,30 @@ private function invokeMethod(&$object, $methodName, array $parameters = array()
         $method->setAccessible(true);
         return $method->invokeArgs($object, $parameters);
     }
+
+    /**
+     * @dataProvider validMoveProvider
+     */
+    public function testValidMove(array $move, bool $isValid): void
```

Comment:
This test passes in latest develop, so it's not really showing that this fixes the deprecation warning in PHP7.2+

Reply:
i think it's because of the setup you guys have on chess.com
on chesskid, this is failing

Reply:
```
1) Chesskid\Tests\Unit\ChessGame\DataTransformer\SanMoveListToPiotrTransformerTest::testTransformStandardChess with data set #0 (Chesskid\Core\Entity\ChessGame\SanMoveList Object (...), 'kA0Kow3N')
The each() function is deprecated. This message will be suppressed on further calls

/www/sites/www.chesskid-dev.com/current/vendor/chesscom/chess-game/src/Chess/Game/ChessGame.php:2293
/www/sites/www.chesskid-dev.com/current/src/Chesskid/Core/Entity/Game/ChessGame.php:339
/www/sites/www.chesskid-dev.com/current/src/Chesskid/Core/Service/ChessGame/DataTransformer/SanMoveListToPiotrTransformer.php:62
/www/sites/www.chesskid-dev.com/current/src/Chesskid/Core/Service/ChessGame/DataTransformer/SanMoveListToPiotrTransformer.php:31
/www/sites/www.chesskid-dev.com/current/src/Chesskid/Tests/Unit/ChessGame/DataTransformer/SanMoveListToPiotrTransformerTest.php:29
/www/sites/www.chesskid-dev.com/current/src/Chesskid/Main/MainTest.php:65
/www/sites/www.chesskid-dev.com/current/src/Chesskid/Main/MainTest.php:18
```

Reply:
The test is not needed. Develop already fails in latest develop as we can see in this link: https://travis-ci.org/ChessCom/Chess-Game/jobs/476987303